audio/audacity/Portfile

```diff
@@ -63,10 +64,29 @@ homepage            http://www.audacityteam.org/
 
 compiler.blacklist  {*gcc-[234].[0-8]}
 
-wxWidgets.use       wxWidgets-3.0
+if {[variant_exists wx32] && [variant_isset wx32]} {
+    wxWidgets.use   wxWidgets-3.2
+    # we want to use the wxWidgets-3.2 "bleeding edge" port!
+    wxWidgets.port  wxWidgets-3.2
+    patchfiles-append \
+                    patch-wxw-31x.diff
+} else {
+    wxWidgets.use   wxWidgets-3.0
+}
+
 depends_build-append \
                     port:${wxWidgets.port}
 
+if {![catch {set wxw_stdlib [active_variants ${wxWidgets.port} stdlib]} err]} {
+    if {${wxw_stdlib}} {
+        long_description-append "This port needs ${wxWidgets.port} without the +stdlib variant."
+        pre-fetch {
+            ui_error "This port needs ${wxWidgets.port} without the +stdlib variant."
+            return -code error "reinstall port:${wxWidgets.port} without the +stdlib variant"
```

Comment:
Out of curiosity: what problems are caused by `+stdlib`?

Reply:
I honestly don't remember, I ran into that in March when building against wxWidgets-3.0 . 

Is that the wxWidgets +stdlib variant used by many? In that case it might be worth investigating if this is only a build incompatibility.